pkg/api/tag.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+)
+
+type TagResponse struct {
```

Comment:
This type can be unexported and export it only in export_test.go for tests. It also can have a constructor to avoid repeating the assignments in every handler.